### ISSUE TITLE
docs(aio). Structural directives. No path in navigation menu

### ISSUE THREAD

**Issue by Yerkon** [angular/angular#21848]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior

https://angular.io/guide/structural-directives
There is no path to this URL in the **navigation menu**
![2018-01-29_11-05-59](https://user-images.githubusercontent.com/4002847/35494395-8c923740-04e4-11e8-8880-1ec03b7977da.png)
![2018-01-29_11-38-12](https://user-images.githubusercontent.com/4002847/35495048-04485824-04e9-11e8-8460-4a930b9f25ab.png)

## Expected behavior

I should be able navigate to this page from navigation menu, right?. I see only Attribute Directives page

**Comment by Yerkon** [comment]
Can I create PR for updating **navigation.json** 
```
           {
              "url": "guide/dynamic-component-loader",
              "title": "Dynamic Components",
              "tooltip": "Load components dynamically."
            },
            {
              "url": "guide/attribute-directives",
              "title": "Attribute Directives",
              "tooltip": "Attribute directives attach behavior to elements."
            },
          // Here =====>
            {
              "url":"guide/structural-directives",
              "title": "Structural Directives",
              "tooltip": "Manipulate the DOM with structural directives"
            } ,
            {
              "url": "guide/pipes",
              "title": "Pipes",
              "tooltip": "Pipes transform displayed values within a template."
            },
            {
              "url": "guide/animations",
              "title": "Animations",
              "tooltip": "A guide to Angular's animation system."
            }
```

**Comment by trotyl** [comment]
The change was made in https://github.com/angular/angular/commit/53f91189a12ee214d7314ea73ce138dcdb1333c8#diff-85eb1d97b3bd1d4c25b72f947df2185f by @kapunahelewong

**Comment by kapunahelewong** [comment]
Thank you @Yerkon and @trotyl. @trotyl is right, but it's not in master right now, as you've pointed out, @Yerkon though I _think_ it should be. @gkalpak do you know of any reason that we shouldn't have it in the nav?

**Comment by gkalpak** [comment]
No, I don't know of any reason. Seems like an accidental removal (unrelated to the NgModules PR).
So, yes, we should add it back :smiley:

@Yerkon, thanks for catching this. Feel free to submit a PR adding it back 👍

**Comment by trotyl** [comment]
Closed by #21856
